spec/models/comment_spec.rb

```diff
@@ -2,8 +2,7 @@
 
```

Comment:
- [ ] In your project, you have file with '0' as the name. Please update it so it looks professional. 

![image](https://user-images.githubusercontent.com/58504655/229030663-365716d5-912f-4ed3-baa5-8f99275a2990.png)

Reply:
Thank you for reviewing the project. This has been resolved